config/routes.rb

```diff
@@ -1,10 +1,34 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      get 'comments/index'
+      get 'comments/create'
```

Comment:
- [ ] Great job working on this milestone. I think to add/create a comment, we have to use a POST method and not GET as you are currently doing. Kindly review your method